src/api/app/assets/stylesheets/webui/build-results.scss

```diff
@@ -66,6 +70,8 @@
   .build-info-content {
     @extend .py-2;
     @extend .px-3;
+    padding-top: 0.4rem !important;
+    padding-bottom: 0.4rem !important;
```

Comment:
You already have py-2 above, it's one or the other

Reply:
I'll remove the `@extend .py-2`, this is the culprit of an issue, visually it shows little dots below each row for the archs results, those little dots just aren't dots... are parts of the text that are hidden in the question mark icon :smile:

Reply:
I would probably just move the text from that collapse lower then, but I guess it doesn't matter either way